### PR TITLE
feat: added wishlist page

### PR DESCRIPTION
**Pull request by sks-21**
Created the **wishlist page** to display the favourite items of the use

Screen Preview
kickKart-wishlist

### CODE REVIEW

File: wishlist/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+.card-container {
+    padding: 0rem 5rem;
+    justify-content: center;
+}
+
+.overlay-btn-heart {
+    background-color: var(--color-error);
+    color: var(--color-white);
+}
```
Comment (KanekarSnehal) [inline wishlist/wishlist.css]: add new line at the end of code file
Refer here: https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log

File: wishlist/wishlist.html [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+    <link rel="stylesheet" href="/wishlist/wishlist.css">
+</head>
+
+<body>
+
+    <!-- || Navbar  -->
+    <nav class="ecom-nav nav flx-row flx-main-center flx-space-between flx-wrap">
+        <a href="/index.html" class="flx-row flx-center ecom-title-container">
+            <div class="navbar-logo">
+                <img src="/assets/images/img-logo.png" alt="" loading="eager" class="navbar-logo-img" />
```
Comment (KanekarSnehal) [inline wishlist/wishlist.html]: alt attribute is empty

File: wishlist/wishlist.html [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+            </div>
+        </div>
+        <div class="vertical-card">
+            <div class="vertical-card-overlay">
+                <div class="overlay-btn">
+                    <button class="overlay-btn-heart"><i class="far fa-heart"></i></button>
+                </div>
+            </div>
+            <div class="content-img-container">
+                <img class="content-img" src="/assets/images/card-dress.jpeg" alt="" />
```
Comment (KanekarSnehal) [inline wishlist/wishlist.html]: here also alt is empty

File: wishlist/wishlist.html [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
```
Comment (KanekarSnehal) [inline wishlist/wishlist.html]: alt is empty

File: wishlist/wishlist.html [hunk 5]
```diff
@@ -0,0 +1,198 @@
...
+                </button>
+            </div>
+        </div>
+    </div>
+
+
+
+</body>
+
+</html>
```
Comment (KanekarSnehal) [inline wishlist/wishlist.html:195]: add new line same as above

**Review by KanekarSnehal — COMMENTED** [review]
Good work and good naming conventions followed, just make sure alt attribute is not empty and new line at end of file.

**Review by aryanandarnav13 — APPROVED** [review]

**Review by neetukumari4858 — APPROVED** [review]
You did good Work It's Looking good just do some changes which other have Mentioned

**Pull request merged**